examples/dev/slicing/random_points.py

```diff
@@ -0,0 +1,20 @@
+import argparse
+from skimage import data
+import numpy as np
+import napari
+
+"""
+Stress the points layer by generating a large number of points.
+"""
+
+parser = argparse.ArgumentParser()
+parser.add_argument("n", type=int, default=10_000_000)
```

Comment:
```suggestion
parser.add_argument("-n", type=int, default=10_000_000)
```

There might be a more explicit way to do this, but this is one way to make `n` an optional argument. Right now, it is a required argument (for Python 3.9.6 anyway) even though you provided a default, so I can't run this without specifying `n`.

Reply:
I fixed it with nargs='?' instead. I definitely assumed the behavior of 'n' and '-n' would be the same. Whoops!